pkg/rancher-desktop/main/deploymentProfiles.ts

```diff
@@ -21,14 +21,7 @@ export class DeploymentProfileError extends Error {
  * Lockable default settings used for validating deployment profiles.
  * Data values are ignored, but types are used for validation.
  */
-export const lockableDefaultSettings = {
-  containerEngine: {
-    allowedImages: {
-      enabled:  true,
-      patterns: [] as Array<string>,
-    },
-  },
-};
+export const lockableDefaultSettings = settings.defaultSettings;
```

Comment:
Shouldn't we get rid of this variable (and the comment above), given that from now on all settings should always be lockable?

The variable is only used once, so just replace its usage with `settings.defaultSettings` and get rid of the alias.

Reply:
I am fine with this. I originally kept it around in the event that we might need to filter the list, but we can always reintroduce the variable if we need to do so.